app/resources/numismatic_issues/numismatic_issue_decorator.rb

```diff
@@ -135,4 +135,16 @@ def state
   def subjects
     numismatic_subject.map { |s| s.decorate.title }
   end
+
+  def metric_labels_first
+    ["Accessions", "Firms", "Monograms"]
+  end
+
+  def metric_labels_second
+    ["People", "Places", "References"]
+  end
+
+  def metric_issues_label
+    ["Issues"]
+  end
```

Comment:
See the comments in `numismatics.html.erb`. I don't think the NumismaticIssueDecorator is the right place for these.